pages/index.js

```diff
@@ -15,6 +18,23 @@ export default function Home() {
           </Heading>
         </main>
 
+        {!session && (
+          <>
+            Not signed in <br />
+            <button onClick={() => signIn()}>Sign in</button>
+          </>
+        )}
+        {session && (
+          <>
+            <h2>Signed in as {session.user.email}</h2> <br />
+            <button onClick={() => signOut()}>Sign out</button>
+          </>
+        )}
+        <main className={styles.main}>
+          <h1 className={styles.title}>Welcome to JAM</h1>
+          <a href="/dashboard">Click here for the Admin Dashboard</a>
+        </main>
```

Comment:
No more `styles.main` after the changes here: https://github.com/redbadger/jams/pull/48 😢 . Also it looks like this block is repeated with the one on lines 15-19, maybe merge these together using the Chakra components?

Reply:
this looks kinda like a merge slip up on my side, I got rid of all the styles 🤔

Reply:
yeah that was it, forgot to commit the cleaned up file after rebase